webapp/src/components/Tooltip/index.js

```diff
@@ -10,7 +10,7 @@ import styles from './styles'
 
 const useStyles = makeStyles(styles)
 
-const Tooltip = ({ anchorEl, anchorOrigin, open, onClose, children }) => {
+const Tooltip = ({ anchorEl, anchorOrigin, open, onClose, children}) => {
```

Comment:
why modify this line?

Reply:
It was due to when I was testing if tooltip was the best way to display the `onmouseover` event, I've reverted the changes